### PR TITLE
sort species alphabetically

### PR DESCRIPTION
**Pull request by ens-st3**

### CODE REVIEW

**Comment by thomasmaurel** [comment]
@ens-st3: I am afraid that we can't accept this PR. We can't really allow the script to change @INC as it could have bad consequences downstream. Also everybody's Git repository setup is different, this change is making assumptions about the directory structure. This script expects you to have to have these repos in the PERL5LIB variables (I should add this requirement to the script).

Ordering reports by species alphabetically is a good idea. Also I was thinking that maybe it would be nice to print the content of the file on the screen as well. Please feel free to create a new PR for ordering species or to create a JIRA ticket for me to look at these.

**Comment by ens-st3** [comment]
Can you accept the second commit or do you need that as a separate pull request ?

**Comment by thomasmaurel** [comment]
@ens-st3: Could you please make a new PR?

**Comment by ens-st3** [comment]
I've reverted the first commit and updated the title. Can you accept like this ?

File: misc_scripts/report_genomes.pl [hunk 1]
```diff
@@ -200,7 +200,9 @@ =head1 VERSION
 my $dbs = {};
 my $species = {};
 $logger->info("Comparing releases");
-while (my ($set_chain, $genome) = each %{$genomes}) {  
+my @set_chains = sort keys %{$genomes};
+foreach my $set_chain (sort keys %{$genomes}) {
```
Comment (marcoooo) [inline misc_scripts/report_genomes.pl]: dummy question: why don't you loop over your newly created @set_chains var?

**Pull request merged**